dashboard/app/controllers/musiclab_controller.rb

```diff
@@ -22,15 +23,31 @@ def gallery
       to_json
   end
 
+  # Hard-coded list of channel IDs for the mini player if DCDO flag is set to `true`
+  CHANNELS = %w(
+    syBuoFelbGB3eOmNVoQGrWMXEk0l1EhkIX6c08ujq6s
+    Ehnks69B0Whcn_YQQNCK4GUHAPU3WSG2jfilvQF1kfo
+    6Xc53NIhwFxjjwsSaoj_eSiRbbXr97BYQ3W_7vIaAwY
+    NwTkJSskTswEOtgy6TbaJ-8SonhhSxojrJjlJLBko4w
+    PCO7mvB5ylByrbpF7tEbzXdYOqrALhW3M5OlcDRIF7E
+  )
+
+  # GET "/musiclab/embed"
   def embed
     response.headers['X-Frame-Options'] = 'ALLOWALL'
     response.headers['Content-Security-Policy'] = ''
 
     view_options(no_header: true, no_footer: true, full_width: true, no_padding_container: true)
 
-    channel_ids = params[:channels] ? params[:channels].split(',') : []
+    channel_ids_from_featured_projects = ProjectsList.fetch_active_published_featured_projects('music')["music"].map {|project| project['channel']}
+    if get_channel_ids_from_constant?
+      channel_ids_from_featured_projects = CHANNELS
+    end
+    channel_ids_from_params = params[:channels].nil? ? [] : params[:channels].split(',')
+    all_channel_ids = channel_ids_from_params.empty? ? channel_ids_from_featured_projects : channel_ids_from_params
+    selected_channel_ids = all_channel_ids.sample(NUM_MINI_PLAYER_PROJECTS)
```

Comment:
Saw your note about needing 5 featured projects before merging...just confirming this will work with less than 5, just will look a bit odd b/c it won't fill up the UI?

Reply:
That's correct.